examples/param_est.ipynb

```diff
@@ -0,0 +1,234 @@
+    "outputs = [\n",
+    "    {'x': 1.83},\n",
+    "    {'x': 36.95},\n",
+    "    {'x': 62.36},\n",
+    "    {'x': 77.81},\n",
+    "    {'x': 83.45},\n",
+    "    {'x': 79.28},\n",
+    "    {'x': 65.3},\n",
+    "    {'x': 41.51},\n",
+    "    {'x': 7.91},\n",
+    "]"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "For this example, we will define specific parameters that we want to estimate.\n",
+    "\n",
+    "We can pass the desired parameters to our __keys__ keyword argument."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "keys = ['thrower_height', 'throwing_speed']"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To really see what `estimate_params()` is doing, we will print out the state before executing the estimation"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Printing state before\n",
+    "print('Model configuration before')\n",
+    "for key in keys:\n",
+    "    print(\"-\", key, m.parameters[key])\n",
+    "print(' Error: ', m.calc_error(times, inputs, outputs, dt=1e-4))"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now, we will run `estimate_params()` with the data."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "m.estimate_params([(times, inputs, outputs)], keys, dt=0.01)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now, let's see what the new parameters are after estimation."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print('\\nOptimized configuration')\n",
+    "for key in keys:\n",
+    "    print(\"-\", key, m.parameters[key])\n",
+    "print(' Error: ', m.calc_error(times, inputs, outputs, dt=1e-4))"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Sure enough- parameter estimation determined that the thrower's height wasn't 20 m, instead was closer to 1.9m, a much more reasonable height!"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "`estimate_params()` creates a structure of 'runs' by constructing each index of times, inputs, and outputs and placing them into a tuple.\n",
+    "\n",
+    "                `runs = [(t, u, z) for t, u, z in zip(times, inputs, outputs)]`\n",
+    "\n",
+    "\n",
+    "Using our optimization function, which runs `calc_error()` as a subroutine (more information about `calc_error()` found in our Calculating Error Example), given each of the runs.\n",
```

Comment:
This sentence doesn't make sense to me. Can we reword it?

Reply:
As discussed in our meeting, let's create a new example with multiple runs to introduce the concept of runs. I have ideas for this, so let me know if you need help. 

Then we can reword this to just describe how calc_error is applied to each run independently, without the definition of runs.